e2e/cypress/integration/channel/archived_channels_spec.js

```diff
@@ -156,4 +158,249 @@ describe('Leave an archived channel', () => {
+
+        // # Create private channel
+        createPrivateChannel(testTeam.id).then((channel) => {
+            archivedPrivateChannel1 = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${archivedPrivateChannel1.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+        });
+
+        // # Create another private channel
+        createPrivateChannel(testTeam.id).then((channel) => {
+            archivedPrivateChannel2 = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${archivedPrivateChannel2.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+
+            // # Leave the channel
+            cy.uiLeaveChannel();
+        });
+
+        // # Leave channel modal is visible
+        cy.get('#confirmModal').should('be.visible');
+        cy.get('#confirmModalButton').click();
+
+        // # Click on add channel button
+        cy.get('.AddChannelDropdown_dropdownButton').click();
+
+        // # Click on browse channels
+        cy.get('#showMoreChannels').click();
+
+        // # More channels modal opens
+        cy.get('.more-modal').should('be.visible').within(() => {
+            // # Show public channels is visible by default
+            cy.findByText('Show: Public Channels').should('be.visible').click();
+
+            // # Go to archived channels
+            cy.findByText('Archived Channels').click();
+
+            // # Channel list should contain only the private channel user is a member of
+            cy.get('#moreChannelsList').should('contain', archivedPrivateChannel1.display_name);
+            cy.get('#moreChannelsList').should('not.contain', archivedPrivateChannel2.display_name);
+        });
+    });
+
+    it('MM-T1703 - User can open archived channels', () => {
+        let archivedChannel;
+
+        // # Create a public channel
+        cy.apiCreateChannel(testTeam.id, 'channel', 'channel').then(({channel}) => {
+            archivedChannel = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${archivedChannel.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+        });
+
+        // # Click on add channel button
+        cy.get('.AddChannelDropdown_dropdownButton').click();
+
+        // # Click on browse channels
+        cy.get('#showMoreChannels').click();
+
+        // # More channels modal opens and lands on public channels
+        cy.get('#moreChannelsModal').should('be.visible').within(() => {
+            cy.findByText('Show: Public Channels').should('be.visible').click();
+
+            // # Go to archived channels
+            cy.findByText('Archived Channels').click();
+
+            // # More channels list should contain the archived channel
+            cy.get('#moreChannelsList').should('contain', archivedChannel.display_name);
+        });
+    });
```

Comment:
I'm getting failing tests here, probably due to sysadmin's user preference. If possible, after config or other requests, please use regular user instead of sysadmin. Also, does the config change at MM-T1696 applicable to the succeeding tests. If yes, then I suggest to move all new tests to new spec file and have the config change in the `before` hook.
![Screen Shot 2020-12-19 at 3 56 02 AM](https://user-images.githubusercontent.com/5334504/102656044-430d6b80-41ae-11eb-81c1-3b85d0e918aa.png)
![Screen Shot 2020-12-19 at 3 56 39 AM](https://user-images.githubusercontent.com/5334504/102656064-4a347980-41ae-11eb-8424-2aa86a942911.png)

Reply:
@saturninoabril  
> Also, does the config change at MM-T1696 applicable to the succeeding tests. If yes, then I suggest to move all new tests to new spec file and have the config change in the before hook.

No, we only need it for that one test case to confirm that when archived channels feature is disabled users cannot viewthem.